config/src/main/java/com/epam/aidial/core/config/AccessControlled.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.aidial.core.config;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import lombok.Data;
+
+import java.util.Collection;
+import java.util.Set;
+
+@Data
+public abstract class AccessControlled {
```

Comment:
May be `RoleBasedEntity`? or just `BaseEntity`

Reply:
renamed to `RoleBasedEntity`